lib/urn.rb

```diff
@@ -1,5 +1,6 @@
 class URN
-  PATTERN = 'urn:(?!urn:)[a-z0-9\-]{1,31}:[\S]+'.freeze
+  PATTERN = %(urn:(?!urn:)[A-Za-z0-9][A-Za-z0-9\-]{1,31}:) +
```

Comment:
Yup: I'll update it now (and add a test to forbid uppercase `urn` as a namespace identifier).